### ISSUE TITLE
Making vector tile feature properties available to feature info templates

### ISSUE THREAD

**Issue by tephenavies** [TerriaJS/terriajs#2612]
### Feature info template for testing
```
#start={"version":"0.0.05","initSources":[
{
  "catalog": [
    {
      "name": "props-test",
      "type": "csv",
      "data": "sa4_code_2011,val,time\n101,2,2016-07-23\n102,3,2016-07-23\n101,4,2016-08-06\n102,5,2016-08-06",
      "featureInfoTemplate": {
        "name": "SA4 Code: {{SA4_CODE11}}",
        "template": "<h3>Property from CSV</h3><p>Value: <b>{{val}}</b></p><h3>Property from vector tile</h3><p>Area: <b>{{ALBERS_SQM}}</b></p>"
      },
      "isEnabled": true
    }
  ]
}
]}
```

TerriaJS branch `vt_feature_info`. This will work with any instance that uses vector tiles.

### Problem
Using the above, properties available to feature info template are like this:
```
{
    "asset_name_from_vector_tiles": "something",
    "available_capacity_from_vt": "[{\"year\":2016,\"capacity\":55,\"load\":41.6,\"available_capacity\":13.4}]",
    "csv_data_attribute": 43.7
}
```
and I don't think mustache can access the nested stringified data.

### Possible solutions:
* Add a JSON.parse function to mustache
    * Is this a security problem?
* Preprocess the nested attributes to be in the form needed by `<chart>` and `<table>`
    * Bad if you want to change the graph or chart (you’ll need to redeploy the vector tile layer)
* Flatten the nested JSON into separate attributes
    * Potentially large number of attributes (is this the currently used solution? I can't read the existing mustache templates well)
* Have a special TerriaJS property (like `terriaProperties`) in vector tile features that is `JSON.parse`d before merging with data properties
    * This means that currently existing properties don’t all become available to feature info templates

## Actual questions:
* Can mustache already decode stringified JSON?
  * Otherwise, do we want to add a `JSON.parse` mustache function?
* Do we want all vector tile properties avaialble for feature info templates?
  * My vote: Yes

**Comment by tephenavies** [comment]
@kring I included you in this in case you have an opinion on either of the "Actual questions" I included at the end.

**Comment by meh9** [comment]
For the Actual Question 2, yes I think we should expose all props.

**Comment by kring** [comment]
Fixed by #2696.
